examples/universal-connector/src/core/index.ts

```diff
@@ -0,0 +1,129 @@
+import type { ProposalTypes, SessionTypes } from '@walletconnect/types'
+import { UniversalProvider } from '@walletconnect/universal-provider'
+
+import type { CreateAppKit } from '@reown/appkit'
+import type { CaipNetwork, InferredCaipNetwork } from '@reown/appkit-common'
+import { AppKit, type Metadata, createAppKit } from '@reown/appkit/core'
+
+const suiMainnet = {
+  id: 784,
+  chainNamespace: 'sui' as const,
+  caipNetworkId: 'sui:mainnet',
+  name: 'Sui',
+  nativeCurrency: { name: 'SUI', symbol: 'SUI', decimals: 9 },
+  rpcUrls: { default: { http: ['https://fullnode.mainnet.sui.io:443'] } },
+  connectParams: {
+    methods: ['sui_signPersonalMessage'],
+    chains: ['sui:mainnet'],
+    events: []
+  }
+}
+
+const suiTestnet = {
+  id: 784,
+  chainNamespace: 'sui' as const,
+  caipNetworkId: 'sui:testnet',
+  name: 'Sui',
+  nativeCurrency: { name: 'SUI', symbol: 'SUI', decimals: 9 },
+  rpcUrls: { default: { http: ['https://fullnode.testnet.sui.io:443'] } },
+  connectParams: {
+    methods: ['sui_signPersonalMessage'],
+    chains: ['sui:testnet'],
+    events: []
+  }
+}
+
+const networks = [
+  {
+    namespace: 'sui',
+    name: 'Sui',
+    methods: ['sui_signPersonalMessage'],
+    chains: [suiMainnet, suiTestnet],
+    events: []
+  }
+]
+
+type ExtendedNamespaces = Omit<SessionTypes.Namespace, 'chains'> & {
+  chains: InferredCaipNetwork[]
+  namespace: string
+}
+
+type Config = {
+  projectId: string
+  metadata: Metadata
+  networks: ExtendedNamespaces[]
+}
+
+class UniversalConnector {
```

Comment:
I'm confused why do we have this defined in this example app where we provide this from `@reown/appkit-universal-connector`? 🤔